content/applications/general/digest_emails.rst

```diff
@@ -0,0 +1,112 @@
+:show-content:
+
+=============
+Digest Emails
+=============
+
+**Digest Emails** are periodic snapshots sent to your organization via email that include
+high-level information about how your business is performing.
+
+Navigate to Digest Emails by going to 
+:menuselection:`Settings --> General Settings --> Statistics`, then activate the **Digest Emails**
+feature and click on save.
+
+.. image:: digest_emails/digest-email-settings.png
+   :align: center
+   :alt: Digest Emails section inside General Settings.
+
+You can control a variety of settings for your Digest Emails, such as:
+
+- which KPIs are shared in the Digest
+- how often Digest Emails are sent
+- who in your organization receives Digest Emails
+- creating custom Digest Email templates
+- adding additional KPIs (Studio required)
+
+.. note::
+  By default, Digest Email is *enabled*, and *Your Odoo Periodic Digest* serves as the primary
+  template, which includes all KPI measurements across your Odoo database and is sent daily to
+  administrators.
+
+.. _digest-emails/customize-digest:
+
+Customize *Your Odoo Periodic Digest*
+=====================================
+
+To cuzomize, go to the same **Digest Email** section inside **General Settings**, and then click the **[-->]** icon next to the dropdown selection with *Your Odoo Periodic Digest* selected in the dropdown menu.
+
+A popup window will appear and present a variety of editable settings, which include:
+
+- **Digest Title** - what you would like your Digest Email to be called
+- **Periodicity** - control the regimen in how often Digest Emails are sent
+- **KPIs** - check/uncheck each calculated KPI that will appear in Digest Emails
+- **Recipients** - add/remove users who receive your Digest Emails
+- **Custom** - add your own KPIs (Studio required)
+
+.. image:: digest_emails/periodic-digest.png
+   :align: center
+   :alt: Customize default Digest Email settings and custom KPIs.
+
+.. _digest-emails/custom-emails:
+
+Create custom digest emails
+===========================
```

Comment:
Verbs such as "create", "manage" etc are not often necessary.
https://www.odoo.com/documentation/saas-15.1/contributing/documentation/content_guidelines.html#titles-and-headings

Don't forget to add a blank line after the title. Actually, between all different commands. Please adapt the rest of the doc accordingly.

```suggestion
Custom digest emails
====================

```